spec/features/system_admin/users/search_users_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Search users" do
+  context "as a system admin" do
+    let(:user) { create(:user, system_admin: true) }
+    let(:second_user) { create(:user, system_admin: true) }
+    let(:lead_school) { create(:school, :lead) }
+    let(:provider) { create(:provider) }
+
+    before do
+      user.lead_schools << lead_school
```

Comment:
Do we need all this setup? The search just searches for users doesn't it by last name or email so we could just do one user without provider or lead school?

Reply:
i think we do need the setup, as this is being rendered on the system admin/users page and so has to be associated with a provider and a school to show up

Reply:
They should show up on there with no school or provider.

Reply:
Ahh. yeah you are correct, the test was failing because I hadn't called the let for the second user, not because the setup was missing

Reply:
Should be fixed now